src/main/flight/rpm_filter.c

```diff
@@ -89,6 +90,7 @@ void rpmFilterInit(const rpmFilterConfig_t *config, const timeUs_t looptimeUs)
     rpmFilter.minHz = config->rpm_filter_min_hz;
     rpmFilter.maxHz = 0.48f * 1e6f / looptimeUs; // don't go quite to nyquist to avoid oscillations
     rpmFilter.fadeRangeHz = config->rpm_filter_fade_range_hz;
+    rpmFilter.fadeRangeHzInv = 1.0f / config->rpm_filter_fade_range_hz;
```

Comment:
This can be undefined, I just realized. `rpm_filter_fade_range_hz` can be 0. The old code protected against division by 0.

Reply:
Good catch, fixed.